compiler/tests/fixtures/stress/tuple_stress.fe

```diff
@@ -0,0 +1,43 @@
+contract Foo:
+    my_sto_tuple: (u256, i32)
```

Comment:
There's this `todo!("encode tuple from storage")` in the analyzer...what exactly is it doesn't yet work with tuples in storage yet. Is it that we  can not yet move an entire tuple from storage to memory?

Reply:
Right now someone would need to do `self.my_sto_tuple.to_mem().abi_encode()` instead of `self.my_sto_tuple.abi_encode()`. It's a low priority `todo` imo, we might not event want to support encoding values from storage.